templates/patient_finder/finder.html.twig

```diff
@@ -44,11 +44,15 @@
                                 </thead>
                                 <tbody>
                                     {% for p in rp %}
-                                        <tr>
```

Comment:
above here can you add a translate to the title text as [pointed out by Luis](https://community.open-emr.org/t/recent-patients-tab-has-empty-date-of-birth-column/21535/4?u=stephenwaite)?
```
                                            <th scope="col">{{ h.title|xlt }}</th>

Reply:
I have made this change.

Reply:
It looks like "Recent Patients" (on the tab) is not being translated.  I looked at the code, and I think it is correct...(this is with my install set for standard French).  Is this something that needs to be fixed with this PR?

![xlate](https://github.com/openemr/openemr/assets/7832037/1c081ca8-65da-43e5-af09-cc164cb69a9d)

Reply:
hi @DrBassman, no need to fix, it's just missing the constant in the translation engine. You can see it work by adding the constant and then load a new definition under Admin->System->Language